src/styles/theme.scss

```diff
@@ -22,6 +22,7 @@ $color-9: #555555 !default;
 $color-10: #f2f2f2 !default;
 $color-11: #999 !default;
 $color-12: #333 !default;
+$color-13: #ffffff80 !default;
```

Comment:
używasz gdzieś? nie widziałam

Reply:
w css'ach jest to w variancie secondary, kolor tego X
<img width="476" alt="Screenshot 2023-10-10 at 11 35 15" src="https://github.com/landingi/ui-kit/assets/132905788/7b787085-b6ba-4403-a6f5-1d0b7cb122f9">